calicoctl-wait/calicoctl-wait.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"time"
+)
+
+func main() {
+	// Spin continously in tight loop
+	for {
+		time.Sleep(500 * time.Millisecond)
```

Comment:
This seems too low, should we just sleep longer?

Reply:
@krishgobinath - I prematurely merged - didn't notice that @doublek was pending. Can you open another PR to address this comment?

Reply:
I had unit tested with 500ms and 1 second, I didn't spot any difference between those 2 in terms of CPU utilization rate. The worst case is if we don't sleep then CPU utilization rate is 100%.

Reply:
@bcreane If required I'll do that.

Reply:
I chatted with @doublek - he's okay skipping this change. So ... never mind.